aea/crypto/base.py

```diff
@@ -167,6 +167,17 @@ def is_transaction_valid(
         :return: True if the random_message is equals to tx['input']
         """
 
+    @staticmethod
+    @abstractmethod
+    def get_contract_address(tx_receipt: JSONLike) -> Optional[str]:
+        """
+        Get the contract address from a transaction receipt.
+
+        :param identifier: the identifier of the ledger
```

Comment:
extra

Reply:
The identifier argument is not present, just remove the docstring line 176